### PR TITLE
presentations/archive/dasharo-onboarding.md: fix updated  osfv-cli setup guide and command syntax

### PR DESCRIPTION
**Pull request by mysteriousCzrsX**
Updated osfv-cli setup guide and commands syntax to comply with current versions of osfv-cli.

### CODE REVIEW

**Review by wiktormowinski — APPROVED** [review]

**Comment by pietrushnic** [comment]
@macpijan I wonder if this is a good place to provide such documentation. Also, do we have to wait for your approval?

**Pull request merged**
